### PR TITLE
Edit and format the RSA Key Generation topic

### PR DESCRIPTION
**Pull request by vnikolova**
The RSA Keys Generation topic was contributed [KTOR-6641](https://youtrack.jetbrains.com/issue/KTOR-6641/Web-feedback-from-JSON-Web-Tokens-Add-a-new-topic-for-RSA-key-generation).

I additionally reworded some of the text and improved the formatting to match our style guide, including:

- Fixed auto-detected formatting issues.
- Added a link to the sample project.
- Added a link summary to the JWT topic (to be displayed in the topic link tooltip).

[Preview on staging](https://ktor.shared.aws.intellij.net/docs/rsa-keys-generation.html)

### CODE REVIEW

File: topics/jwt.md [hunk 1]
```diff
@@ -14,6 +16,10 @@
 <include from="lib.topic" element-id="native_server_not_supported"/>
 </tldr>
 
+<link-summary>
+The %plugin_name% plugin allows you to perform validations of JSON Web Tokens for authentication purposes.
```
Comment (Stexxe) [inline topics/jwt.md:20]: Isn't the purpose to authenticate the clients using JWT?

File: topics/rsa-keys-generation.md [hunk 2]
```diff
@@ -1,91 +1,108 @@
...
-For production use, it is recommended that you choose a more modern alternative, such as <a href="https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm">ES256</a>, that's based on more modern, efficient, and secure cryptography. However, because RSA still has its uses and is supported by the JWT plugin, the documentation below uses RSA.
+For production use, it is recommended that you choose a more modern alternative, such as <a href="https://en.wikipedia.org/wiki/Elliptic_Curve_Digital_Signature_Algorithm">ES256</a>, that's based on more efficient, and secure cryptography compared to RSA.
 </p>
-</tip>
+</warning>
 
 ## Generating an RSA private key
 
-To generate a private key, you can use `OpenSSL`, `ssh-keygen` or any other valid utility. For the rest of this page, OpenSSL will be used.
+To generate a private key, you can use OpenSSL, `ssh-keygen` or any other valid utility. For demonstration purposes,
```
Comment (Stexxe) [inline topics/rsa-keys-generation.md:32]: What does the word **valid** here mean?

**Review by Stexxe — APPROVED** [review]

File: topics/jwt.md (hunk 1 above)
Comment (vnikolova) [reply topics/jwt.md:20]: You're right. I phrased it like that to be more specific but "authenticate clients using JWT" sounds better!

File: topics/rsa-keys-generation.md (hunk 2 above)
Comment (vnikolova) [reply topics/rsa-keys-generation.md:32]: Perhaps better to replace "any other valid utility" with "another tool of your choice for creating authentication key pairs"?

**Pull request merged**